tests/driver_soft_spi/README.md

```diff
@@ -0,0 +1,10 @@
+Expected result
+===============
+The test code transmits a byte and a string over the software SPI MOSI line. This can be verified with a Logic Analyzer connected to the corresponding pins. When you see the clock signal and "0xaa" and "Soft SPI Test String" on the MOSI pin, the test is succesful.
+
+Notes
+==========
+- The test disables all pre-initialized hardware SPI devices
+- The test does not cover MISO inputs
+- The test ignores clock speeds
```

Comment:
So if we ignore clock speeds and doe not cover MISO, there is not much functionality of a SPI bus left, is there?